BUILD.md

```diff
@@ -59,7 +59,7 @@ Install dependencies:
 
 ```sh
 sudo apt install -y podman dh-python build-essential fakeroot make libqt6gui6 \
```

Comment:
fakeroot can be removed now that `Rules-Requires-Root: no` is set (unless something else needs it?)

Reply:
Good call. :+1: